### PR TITLE
Test src dvd as repo if at least sle 12 SP3

### PR DESCRIPTION
**Pull request by SergioAtSUSE**
Wait for tests on local instance for

* opensuse [copland#188#step/zypper_info/18](http://copland.arch.suse.de/tests/188#step/zypper_info/18)
* sle 12 SP2 [copland187#step/zypper_info/8](http://copland.arch.suse.de/tests/187#step/zypper_info/8)
* sle 12 SP3 [copland#194#step/zypper_info/20](http://copland.arch.suse.de/tests/194#step/zypper_info/20)

### CODE REVIEW

File: tests/console/zypper_info.pm [hunk 1]
```diff
@@ -30,11 +30,20 @@ sub run() {
     die "Missing package name. Expected: /$expected_package_name_vim/"
       unless $info_output_vim =~ /$expected_package_name_vim/;
 
-    if (check_var('DISTRI', 'sle')) {
+    my $is_opensuse = check_var('DISTRI', 'opensuse');
+    my $is_at_leat_sle_12_SP3 = check_var('DISTRI', 'sle') && (check_var('VERSION', '12-SP3') || int get_var('VERSION') >= 13);
```
Comment (okurz) [inline tests/console/zypper_info.pm]: typo in variable name. there is the sle_version_at_least helper method

File: tests/console/zypper_info.pm [hunk 2]
```diff
@@ -30,11 +30,20 @@ sub run() {
...
+
+    if ($is_at_leat_sle_12_SP3) {
         # use dvd2 as the src-repository
         zypper_call('ar --disable --type plaindir cd:///?devices=/dev/sr1 repo-source');
     }
 
+    if ($is_opensuse || $is_at_leat_sle_12_SP3) {
```
Comment (coolo) [inline tests/console/zypper_info.pm]: how about checking if ISO_2 is set and drop the version requirements? Perhaps we want to test zypper info on SP2 maintenance later on.

Comment (SergioAtSUSE) [inline tests/console/zypper_info.pm]: ISO_2 is set up in test module, which is executed by both. so the var ISO_2 would exist in SP2 making the test to fail.

Comment (okurz) [inline tests/console/zypper_info.pm]: I don't understand what you mean with "ISO_2 is set up in test module", where exactly?

Comment (SergioAtSUSE) [inline tests/console/zypper_info.pm]: I mean on openQA settings for the test suite: https://openqa.suse.de/admin/test_suites

Search for `extra_tests_in_textmode`

**Comment by okurz** [comment]
> btw. released SLE12 has source repositories by default installed

but only if the system is registered over SCC which is not the case for all tests as we have them.

**Comment by mimi1vx** [comment]
@okurz  but all QAM test uses registered images ...

**Pull request merged**